server/test/handlers/getAppointments.test.js

```diff
@@ -0,0 +1,80 @@
+const appointmentHandler = require('../../models/handlers/appointment');
+
+describe('testing getting appointment details functionality', () => {
+    beforeEach(() => {
+        jest.resetModules();
```

Comment:
Hi Amy, 
I also tried to manually test the endpoint locally by running npm start in the server folder. I face this error (see the screenshot). Can you suggest a reason for this error?
<img width="984" alt="Screen Shot 2020-10-30 at 10 29 27 AM" src="https://user-images.githubusercontent.com/71143782/97717799-919a6580-1a9b-11eb-9651-c73b92ed3d5e.png">
<img width="978" alt="Screen Shot 2020-10-30 at 10 29 16 AM" src="https://user-images.githubusercontent.com/71143782/97717810-952dec80-1a9b-11eb-894c-3cf5b894d290.png">

Reply:
Hi Nancy,

This may be related to the above issue. I would review and follow the steps in the Local Setup section in the ReadME.md file to ensure that you have a local database set up and that the required environment variables are initialized in the .env and .jest files.